packages/public/server/src/module/User/src/User/Manager/UserManagerInterface.php

```diff
@@ -84,4 +86,15 @@ public function getUserFromAuthenticator();
      * @return mixed
      */
     public function updateUserPassword($id, $password);
+
+    /**
+     * Adds or sets user fields to a user.
+     *
+     * <code>
+     * $userManager->setFields($user, ['foo' => 'bar', 'acme' => 'bar']);
+     * </code>
+     * @param UserInterface $user
+     * @param array $data
+     */
+    public function setFields($user, $data);
```

Comment:
If I understood the implementation correctly: this is more a `mergeFields`, right?

Reply:
Yes, that's true. I'll rename it, because I do think it is the behaviour we want. We might need a function for removing a field though.